src/backend/app/Http/Controllers/Api/v1/OrdersController.php

```diff
@@ -3,30 +3,73 @@
 namespace App\Http\Controllers\Api\v1;
 
 use App\Http\Controllers\Controller;
+use App\Http\Requests\DestroyClient;
+use App\Http\Requests\IndexOrder;
+use App\Http\Requests\StoreClient;
+use App\Http\Requests\StoreOrder;
+use App\Http\Requests\UpdateOrder;
+use App\Models\Client;
+use App\Models\Order;
+use Illuminate\Http\JsonResponse;
 use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Psy\Util\Json;
```

Comment:
toto urcite nepouzivas a je to naviac, prejdi si to pre istotu vsade a vsetko co je unused prec